doc/user/content/sql/system-tables.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "System Tables"
+description: "System tables store metadata about your Materialize instance."
+weight: 14
+menu:
+  main:
+    parent: 'sql'
+---
+
+Like PostgreSQL's `pg_catalog`, Materialize provides a handful of system tables
+containing up-to-date metadata for a given Materialize instance. These tables are
+maintained as [views](../create-view), and can be [queried](../select) as such.
```

Comment:
What do you see as the consequence of these tables being maintained as views from the user's perspective? If there isn't anything really compelling, we might consider eliding that detail.